service/src/test/java/bio/terra/workspace/common/fixtures/ControlledResourceFixtures.java

```diff
@@ -183,6 +179,27 @@ public static ApiAzureVmCreationParameters getAzureVmCreationParameters() {
         .customScriptExtension(getAzureVmCustomScriptExtension());
   }
 
+  public static ApiAzureVmCreationParameters
+      getAzureVmCreationParametersWithEphemeralOsDiskAndCustomData() {
+    return new ApiAzureVmCreationParameters()
+        .name(uniqueAzureName(AZURE_VM_NAME_PREFIX))
+        .region("westcentralus")
+        .vmSize(VirtualMachineSizeTypes.STANDARD_D8S_V3.toString())
+        .vmImage(
+            new ApiAzureVmImage()
+                .publisher("microsoft-dsvm")
+                .offer("ubuntu-1804")
+                .sku("1804-gen2")
+                .version("latest"))
```

Comment:
We should probably make it a config parameter, so that it's easier to update later.

Reply:
I am just copying what the other tests do